hash/snes.xml

```diff
@@ -51261,6 +51745,19 @@ to ensure nothing has been touched in the Retro Quest cart production in 2013/20
 		</part>
 	</software>
 
+	<software name="nightbst">
+		<description>Nightmare Busters (USA)</description>
+		<year>2013</year>
+		<publisher>Super Fighter Team</publisher>
+		<info name="developer" value="Arcade Zone" />
+		<part name="cart" interface="snes_cart">
+			<feature name="slot" value="lorom" />
+			<dataarea name="rom" size="2097152">
+				<rom name="Nightmare Busters (USA) (Unl).sfc" size="2097152" crc="abaaff79" sha1="65d156bcc43149d4828289bc78b2565a674f58be" offset="0x000000" />
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Is this game actually related to the Nichibutsu prototype Nightmare Busters, or not?  I get the impression it’s a new game developed by Arcade Zone and published by Super Fighter Team.

If it isn’t the same game, it shouldn’t have a parent/clone relationship with the Nichibutsu prototypes.  If this is the case, change the other Nichibutsu prototype back to `nightbst` and come up with a different short name for this one.

If this is actually an unlicensed hack of the Nichibutsu prototypes, it shouldn’t be the parent, it should be a clone.

Reply:
Apparently the original publisher canceled the release, then the dev team ported the game to phones. Super Fighter Team (Brandon Cobb) found it, then had them finish the SNES game to publish it legally as a third party aftermarket cart.

So in my opinion, this should be the parent, since it's a legal release, sanctioned by the original developers.

Reply:
Hmm, looking into it, it does seem to be built with a license for the original game design and artwork.